target/linux/bcm53xx/image/Makefile

```diff
@@ -194,13 +194,13 @@ define Device/buffalo-wzr-900dhp
   BUFFALO_TAG_PLATFORM := bcm
   BUFFALO_TAG_VERSION := 9.99
   BUFFALO_TAG_MINOR := 9.99
-  IMAGES += factory-DHP-EU.bin factory-DHP2-JP.bin
-  IMAGE/factory-DHP-EU.bin := \
+  IMAGES += DHP-EU-factory.bin DHP2-JP-factory.bin
```

Comment:
So this means here actually two devices (DHP and DHP2) have the same definition, same sysupgrade, but different factory files?
I'm not a fan of moving the additional information in front of "factory" here. Since this is _one_ device, at least from the view of OpenWrt, those are variants of the _image_, not of the device, and thus they should come after the image type. (Otherwise they would most probably be detected as different devices by your image selector, which is only a good idea on first sight, as this will create an impression not matching the reality in OpenWrt).

Despite, if we change stuff anyway, I'd go for all-lower-case as with the "modern" device (image) names.